src/OrchardCore/OrchardCore.Mvc.Core/FileProviders/FileProviderExtensions.cs

```diff
@@ -47,7 +48,7 @@ public static class FileProviderExtensions
                 }
                 else if (inViewsFolder)
                 {
-                    if (extensions.Contains(Path.GetExtension(content.Name)))
+                    if (Array.IndexOf(extensions, Path.GetExtension(content.Name)) != -1)
```

Comment:
Contains goes to LINQ, that's a common pitfall in hot paths. From debugger it seemed that there was only one item so enumerating is overkill.

Reply:
So, in hot paths better to not use LINQ, particularly if the collection has few items.

Here, currently we can have 2 extensions `.cshtml, .liquid`, but normally no more.

Reply:
.NET Core has made things better with LINQ, but there's always an overhead. Especially when check turn into enumeration (`GetEnumerator()`). [Here's some opinionated thoughts about LINQ](https://blog.lahma.net/2018/10/optimizing-jint-part-3-banishing-linq.html) 😉 

LINQ in hot paths is usually bad, but it usually makes code clearer so go with it when it suits.